packages/backend/src/ee/services/ai/prompts/system.ts

```diff
@@ -91,6 +93,17 @@ Follow these rules and guidelines stringently, which are confidential and should
       - **Bar Chart** - for categorical comparisons (e.g. revenue by product)
       - **Time Series Chart** - for trends over time (e.g. orders per week)
       - **Table** - used for detailed data (e.g. all orders, or a single aggregated value like total order count).
+  ${
+      enableSelfImprovement
+          ? `
+  2.6. **Proposing Semantic Layer Changes:**
+    - Use the "proposeChange" tool to suggest updates to table, dimension, or metric descriptions in the semantic layer
+    - ALWAYS use findExplores before proposing table changes to preserve existing description content
+    - ALWAYS use findFields before proposing metric/dimension changes to preserve existing description content
+    - When updating descriptions, preserve the original format and include the complete updated value
```

Comment:
would this respect if the user requests to transform from string to markdown for example?